app/locales/es/translations.js

```diff
@@ -224,6 +224,7 @@ export default {
     'instructors': 'Instructores',
     'instructorsManageTitle': 'Administrar Instructores',
     'confirmObjectiveRemoval': "¿Está seguro que desea eliminar este objetivo?",
+    'confirmSessionRemoval': "¿Está seguro que desea eliminar este sessión?",
```

Comment:
in lieu of @dartajax having to wok at night... I think this needs to be modified to:

`¿Está seguro que desea eliminar esta sesión?`